tests/coverage.sh

```diff
@@ -10,11 +10,11 @@ fi
 
 # Build a Docker image on top of kcov with our scripts
 cat <<EOF | docker build --force-rm -t githooks:coverage -f - .
-FROM kcov/kcov:v33
+FROM kcov/kcov:v36
 
 RUN echo 'deb http://deb.debian.org/debian stretch main' >> /etc/apt/sources.list \
     && (apt-get update || true) \
-    && apt-get install -y git
+    && apt-get install -y git python
```

Comment:
What needs Python here?

Reply:
Not sure, didnt check, the container did not build without...